bedrock/products/templates/products/relay/faq.html

```diff
@@ -80,7 +80,7 @@ <h3> {{ ftl('faq-question-mozmail-question-2') }}</h3>
         </section>
         <section class="mzp-c-details">
           <h3> {{ ftl('faq-question-attachments-question') }}</h3>
-          <p> {{ ftl('faq-question-attachments-answer-v2', size=settings.RELAY_EMAIL_SIZE_LIMIT_KB) }} </p>
+          <p> {{ ftl('faq-question-attachments-answer-v2', size=settings.RELAY_EMAIL_SIZE_LIMIT_MB) }} </p>
```

Comment:
**Nit:** This is just displaying `{size}` in the HTML even with the reference to `settings/base.py`?
![Screenshot 2023-07-28 at 3 49 10 PM](https://github.com/mozilla/bedrock/assets/42309026/4b2f88ca-ebb8-42c8-93ec-555cffe76217)